src/platforms/stm32/src/lib/stm_sys.h

```diff
@@ -0,0 +1,109 @@
+/* This file is part of AtomVM.
+ *
+ * Copyright 2023 Winford (Uncle Grumpy) <winford@object.stream>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
+ */
+
+#ifndef _STM_SYS_H_
+#define _STM_SYS_H_
+
+#include <libopencm3/stm32/rcc.h>
```

Comment:
Why do we need to include rcc.h here?

Reply:
We do not. This is another artifact I missed, that was introduced while testing this PR with the gpio driver I am working on.

Reply:
removed.